db/migrate/20200918181712_create_terms_vocabularies_join_table.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CreateTermsVocabulariesJoinTable < ActiveRecord::Migration[6.0]
+  def change
+    create_join_table :terms, :vocabularies do |t|
```

Comment:
Better use a composite index on both columns, which also should be unique in order to avoid duplicates.